modules/windows-node-services.adoc

```diff
@@ -19,8 +19,8 @@ The following Windows-specific services are installed on each Windows node:
 |Container Network Interface (CNI) plug-ins
 |Exposes link:https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#networking[networking] for Windows nodes.
 
-|Windows Machine Config Bootstrapper (WMCB)
-|Configures the kubelet and CNI plug-ins.
+|Windows Instance Config Daemon (WICD)
+|Creates the Windows worker node. Starts the containerd container runtime and the kubelet on the node. Maintains the state of Windows Services on the node using a config map created by the Windows Machine Config Operator (WMCO).
```

Comment:
WICD basically manages all the services on the Windows instance and does not create the node per se. So I would suggest something like: `Maintains the state of all services running on the Windows instance, defined by WMCO using a ConfigMap, for the instance to function as a worker node`. Definitely needs some word smithing 😄